Packs/Cybereason/Integrations/Cybereason/Cybereason.py

```diff
@@ -1751,21 +1803,25 @@ def login(client: Client):
     return JSESSIONID, creation_time
 
 
-def validate_jsession(client: Client):
-    creation_time = int(time.time())
+def validate_jsession(client: Client, explicit_refresh=False):
+    current_time = int(time.time())
     integration_context = get_integration_context()
-    token = integration_context.get('jsession_id')
-    valid_until = integration_context.get('valid_until')
-    demisto.debug(f"Validate JSESSION: Token is valid until: {valid_until}")
-    if token and valid_until and creation_time < valid_until:
-        demisto.debug(f"Token is still valid - did not expire. token: {token}")
+    token = integration_context.get("jsession_id")
+    valid_until = integration_context.get("valid_until")
+    demisto.debug(f"Validate JSESSION: Token is valid until: {valid_until} and explicit token refresh: {explicit_refresh}")
+    should_refresh = explicit_refresh or not token or not valid_until or current_time >= valid_until
+    demisto.debug(f"Value of Should Refresh: {should_refresh}. Token {token}")
+    if not should_refresh:
+        # Token is still valid and no explicit refresh requested, so use existing token
+        demisto.debug(f"Token is still valid - did not expire. Token: {token}")
```

Comment:
This debug log message prints the session token. While at the debug level, it's best practice to avoid logging sensitive tokens entirely, or to log a masked version of them. It’s recommended to use the `add_sensitive_log_strs` function for handling such cases.